### PR TITLE
az-bicep: add page

### PR DESCRIPTION
**Pull request by piraces**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #6141

### CODE REVIEW

File: pages/common/az-bicep.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# az bicep
+
+> Bicep CLI command group.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Build a Bicep file:
+
+`az bicep build --file {bicep_file}`
```
Comment (blueskyson) [inline pages/common/az-bicep.md:9]:
```suggestion
`az bicep build --file {{path/to/file.bicep}}`
```

File: pages/common/az-bicep.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Attempt to decompile an ARM template file to a Bicep file:
+
+`az bicep decompile --file {json_template_file}`
```
Comment (blueskyson) [inline pages/common/az-bicep.md:13]: Why the description says `ARM template` but the example use `json_template_file`?
```suggestion
- Attempt to decompile an ARM template file to a Bicep file:

`az bicep decompile --file {{path/to/template_file.json}}`
```

File: pages/common/az-bicep.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Install Bicep CLI:
+
+`az bicep install`
```
Comment (blueskyson) [inline pages/common/az-bicep.md:17]: `az bicep install` can be moved to the top because it seems to be the beginning step of this application.

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Here are some suggestions.

File: pages/common/az-bicep.md (hunk 2 above)
Comment (piraces) [reply pages/common/az-bicep.md]: The ARM template is in json... I will change to:
```md
- Attempt to decompile an ARM template JSON file to a Bicep file:
```

**Comment by piraces** [comment]
Commit suggestions

File: pages/common/az-bicep.md (hunk 3 above)
Comment (piraces) [reply pages/common/az-bicep.md:9]: Committed!

File: pages/common/az-bicep.md (hunk 2 above)
Comment (blueskyson) [reply pages/common/az-bicep.md]: I see. Keep the original description is fine.

Comment (piraces) [reply pages/common/az-bicep.md]: Ok. Solved! Thanks!

**Review by blueskyson — APPROVED** [review]

File: pages/common/az-bicep.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Attempt to decompile an ARM template file to a Bicep file:
+
+`az bicep decompile --file {{path/to/template_file.json}}`
+
+- Upgrade Bicep CLI to the latest version:
+
+`az bicep upgrade`
+
+- Show the installed version of Bicep CLI:
```
Comment (navarroaxel) [inline pages/common/az-bicep.md:23]: We use `display` or `print`
```suggestion
- Display the installed version of Bicep CLI:
```

File: pages/common/az-bicep.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`az bicep version`
+
+- List out all available versions of Bicep CLI:
```
Comment (navarroaxel) [inline pages/common/az-bicep.md:27]:
```suggestion
- List all available versions of Bicep CLI:
```

File: pages/common/az-bicep.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# az bicep
+
+> Bicep CLI command group.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Install Bicep CLI:
```
Comment (navarroaxel) [inline pages/common/az-bicep.md:7]: Is there a Bicep non-CLI program/app?

Comment (piraces) [reply pages/common/az-bicep.md:7]: No... there is only an optional VS Code Extension along with the CLI, but no other programs or GUIs.

Comment (piraces) [reply pages/common/az-bicep.md:7]: Also thanks for the suggestions!

Comment (piraces) [reply pages/common/az-bicep.md:7]: @navarroaxel Should I remove the "CLI" word from there? In my personal opinion is relevant and it's always referenced as "Bicep CLI".

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**